src/region/RegionRect.scss

```diff
@@ -7,6 +7,11 @@
 
     position: absolute;
 
+    // stylelint-disable-next-line declaration-no-important
+    box-sizing: content-box !important; // Padding needs to increase the hit area of the button to include the borders
```

Comment:
The other alternative that I considered was to use something like this in `styleShape`, but it seemed somehow worse than just changing the box sizing model:

```
width: `calc(${width}% + ${BORDER_TOTAL}px)`
```